autocorpus/Autocorpus.py

```diff
@@ -378,17 +378,25 @@ def process_files(self):
             raise RuntimeError("A valid config file must be loaded.")
         # handle main_text
         if self.file_path:
-            soup = self.__handle_html(self.file_path, self.config)
-            self.main_text = self.__extract_text(soup, self.config)
-            try:
-                self.abbreviations = Abbreviations(
-                    self.main_text, soup, self.config, self.file_path
-                ).to_dict()
-            except Exception as e:
-                print(e)
+            file_extension = self.file_path.split(".")[-1].lower()
```

Comment:
I don't believe that `self.file_path` is a `pathlib.Path` object, but it _should_ be. That way this can be done the more robust way:
```suggestion
            file_extension = self.file_path.suffix.lower()
```

Reply:
Just in-case, as I this field is used a lot elsewhere, I've done a conversion to Path type just for the type checking function though